packages/core/src/app/payment/paymentMethod/PaypalCommerceCreditDescription.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import { useCheckout } from '@bigcommerce/checkout/payment-integration-api';
+
+import PaymentMethodId from './PaymentMethodId';
+
+const PaypalCommerceCreditDescription: FunctionComponent<{ onUnhandledError?(error: Error): void }> = ({ onUnhandledError }) => {
```

Comment:
Can we move this code to separate package?

Reply:
@animesh1987 I can not move this component to separate packages (paypal-commerce-integration) because it is `scope:integration`. Can we leave it as it is for now?